packages/application-shell/src/components/custom-view-shell/custom-view-shell.tsx

```diff
@@ -94,6 +98,11 @@ function NotificationsContainer(props: TNotificationsContainerProps) {
   );
 }
 
+const ContentWrapper = styled.div`
+  height: 100%;
+  padding: ${designTokens.spacing40} 40px;
```

Comment:
Wondering if we should keep the shell container without margins / paddings.

Users can "easily" do that by using our page components or spacings components.

For instance, it would be tricky then for users to "remove" the double spacing rather than adding one.

Reply:
That's an option, although it would force consumers to use one of our Page Layout components which I was trying to avoid so developers can decide what they want to render as the panel's content.

Reply:
It's the same for Custom Applications, we don't use spacing in the main container.

The issue I see is that when people use the page components they will end up with double spacing, which is impossible for them to get rid of.
On the other hand, if they just render content they will see right away that spacing is missing and they can fix that by adding a container (either a page component or even spacing inset).

Reply:
Yes, you're right we do have the double spacing issue which we have created another task to address it. I was thinking in having some data attribute in the top of the document that page layout component can use to know which css spacing they need to use.

I understand your point although we would also have to think this taking the close icon and the notifications into account.

If this is something you think should block us here, then I'll keep this PR open for a couple of weeks, but we can also try to address one problem at a time, as we were trying to and maybe have a sync with Natalia when she comes back to see how we can improve things?

What do you think?

Reply:
No blocking, we can take it step by step yes.

Let's regroup then when Natalia is back. Thanks